### PR TITLE
Initial setup of server and migration file

### PR DESCRIPTION
**Pull request by Ojself**

### CODE REVIEW

**Comment by Ojself** [comment]
@BlackGhostHD review me, bitte

**Review by BlackGhostHD — APPROVED** [review]
works

File: package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "name": "fastifytest",
```
Comment (janrenz) [inline package.json:2]: please change name

File: package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "version": "1.0.0",
+    "description": "",
+    "main": "app.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "migrate": "node-pg-migrate"
+    },
+    "author": "Ojself",
+    "license": "ISC",
```
Comment (janrenz) [inline package.json:11]: This will be AGPL as well, but can remove this from the package, this will go in a LICENSE File

**Review by janrenz — APPROVED** [review]
changes can be done later together

**Pull request merged**